mkdocs.yml

```diff
@@ -0,0 +1,53 @@
+site_name: HMC
+site_author: Mirantis
+docs_dir: ./docs
+repo_name: GitHub
+repo_url: https://github.com/mirantis/hmc
+copyright:
+  '© 2005 - 2023 Mirantis, Inc. All rights reserved. "Mirantis" and "FUEL" are registered trademarks of Mirantis, Inc. All other trademarks are the property of their respective owners.'
```

Comment:
do we have fresher copyright?

Reply:
updated it with 2024, copied from the mirantis website

Reply:
either javascript or python snippet can be added to make the year = current_year. Please leave it as it is, I'll update when working on the Docs.